content/docs/extensions/neon.md

```diff
@@ -4,11 +4,55 @@ enableTableOfContents: true
 updatedOn: '2024-02-08T15:20:54.277Z'
 ---
 
-With each Neon project, Neon creates  a "neon" extension, which includes functions and views designed to gather Neon-specific metrics. The metrics are intended for use by the Neon team for the purpose of enhancing our service. The views are owned by a Neon system role (`cloud_admin`), but you are able to view them by connecting to the `postgres` database using `psql` and executing the command `\dv neon.*`, as shown below. At present, the extension includes two views for local file cache metrics. We may incorporate additional views in future releases.
+The `neon` extension provides functions and views designed to gather Neon-specific metrics.
 
-```bash shouldWrap
-psql 'postgresql://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/postgres?sslmode=require'
+## The `neon_stat_file_cache` view
 
+Neon computes have a local file cache that extends Postgres shared buffers to approximately 50% of your compute's RAM. The `neon_stat_file_cache` view provides the following data about local file cache usage:
+
+- `file_cache_misses`: The number of times requested data was not found in the local file cache.
+- `file_cache_hits`: The number of times requested data was found in the local file cache.
+- `file_cache_used`: The number of times the local file cache was used.
+- `file_cache_writes`: The number of writes to the local file cache.
+- `file_cache_hit_ratio`: The cache hit ratio for the local file cache. The file cache hit ratio is calculated according to the following formula:
+
+    ```
+    file_cache_hit_ratio = (file_cache_hits / (file_cache_hits + file_cache_misses)) * 100
+    ```
+
+    You can use the cache hit ratio as a measure of requests served from memory rather than Neon's storage layer. For better query performance, frequently accessed data should reside in memory. Generally, for OLTP workloads you should aim for a cache hit ratio of 99% or better. However, the ideal cache hit ratio can depend on the specific workload and data access patterns. In some cases, a slightly lower ratio might still be acceptable, especially if the workload involves a significant amount of sequential scanning of large tables where caching might be less effective.
```

Comment:
```suggestion
    In general, you want a high cache hit ratio. It means more of your requests are served directly from memory rather than the slower storage layer. For OLTP workloads, you should aim for a cache hit ratio of 99% or better. However, the ideal cache hit ratio can depend on your specific workload and data access patterns. In some cases, a slightly lower ratio might still be acceptable, especially if the workload involves a lot of sequential scanning of large tables where caching might be less effective.
```